### DISCUSSION TITLE
Standard set of recipies for the Cookbook?

### DISCUSSION THREAD

**Discussion by demuxer86** [nextcloud/cookbook#2084] [General]
Is anyone sharing their recipe set anywhere?   It would be nice if the cookbook came with a few dozen basic recipes so I don't have to find the easy ones.

**Comment by seyfeb** [comment]
I don't know of anyone. However, with the current system sharing the recipe folder would give access to the recipes. One thing to keep in mind, though, is that recipe references rely on the recipe ids stored in the database so they would certainly break on importing the files.

As for a set of standard recipes. Any set of chosen recipes would be _very_ opinionated in terms of cuisine. But in addition, a single recipe can't be include automatic translations. So one would have to select recipes in multiple languages or fall back to one single language (english).

What I like about the idea, though, is, that this would allow to showcase all cookbook features, maybe even a short tutorial could be run at the beginning to explain the functionality.

**Reply by christianlupus** [reply]
The tutorial, I like very much!

About the recipes' storage location, we will have to discuss things.

Maybe it would be possible to create the recipe on the fly when starting the app for the first time. Then, we could actually use the translation engine, depending on the user's l10n settings.
